WORKSPACE

```diff
@@ -43,3 +43,21 @@ git_repository(
 )
 load("@io_bazel_stardoc//:setup.bzl", "stardoc_repositories")
 stardoc_repositories()
+
+
+load("@graknlabs_bazel_distribution//maven:deps.bzl", assemble_maven_deps = "deps", "maven_artifacts_with_versions")
+assemble_maven_deps()
+
+load("@rules_jvm_external//:defs.bzl", rje_maven_install = "maven_install")
+rje_maven_install(
```

Comment:
we don't need to alias `maven_install` to `rje_maven_install` like in `@graknlabs_dependencies`. here, there are no name confusion and therefore we can keep the original name

Reply:
@lolski addressed in https://github.com/graknlabs/bazel-distribution/pull/293/commits/110644cefa7e51c2cb240377191b20a9444c0fa0